views/components/form-holder/decade-form.pug

```diff
@@ -1,20 +1,17 @@
 li(class="not-active")
-  div(class="collapsible-header valign-wrapper")
-    i(class="small material-icons valign-wrapper") today 
-    | Years
+  div(class="valign-wrapper")
+    i(class="small material-icons valign-wrapper") today
+    |  &nbsp Years
     div(id="decadeDisplay" class="valign-wrapper")
-  div(class="collapsible-body")
-    p
-      div(class="row")
-        div(class="input-field inline col s12 m9 l4")
-          i(class="material-icons prefix") today
-          input(placeholder="Starting Year" id="startingDecade" type="number" min="1940" max="2019")
-      div(class="row")
-        div(class="input-field inline col s12 m9 l4")
-          i(class="material-icons prefix") today
-          input(placeholder="Ending Year" id="endingDecade" type="number" min="1940" max="2019")
-    
-    div(class="as-notes z-depth-1" id="decadeNotesContainer") 
+  div(class="", id="date-box  ")
+    div(class="row")
+      div(class="input-field inline col s12 m9 l4")
+        i(class="material-icons prefix") Start
+        input(placeholder="Starting Year" id="startingDecade" type="number" min="1940" max="2019")
+      div(class="input-field inline col s12 m9 l4")
+        i(class="material-icons prefix") End
+        input(placeholder="Ending Year" id="endingDecade" type="number" min="1940" max="2019")
```

Comment:
[nit] the max should probably be 2020 (or find a way to dynamically fill in the current year)